File: Baekjoon/solved_ac/class2/2609.py
# 2609: 최대공약수와 최소공배수

def gcd(a, b):
  while (b != 0):
    temp = b
    b = a % b
    a = temp

  return a

num1, num2 = list(map(int, input().split()))

GCD = gcd(num1, num2)
LCM = num1 * num2 // GCD

print(f"{GCD}\n{LCM}")

File: Baekjoon/solved_ac/class2/2609.cpp
// 2609 : 최대공약수와 최소공배수

#include <iostream>
using namespace std;

int gcd(int a, int b) {
  while (b != 0) {
    int temp = b;
    b = a % b;
    a = temp;
  }

  return a;
}

int main() {
  int num1, num2;
  cin >> num1 >> num2;

  int GCD = gcd(num1, num2);
  int LCM = (num1 / GCD) * num2;

  cout << GCD << '\n' << LCM;

  return 0;
}